src/main/java/net/dv8tion/jda/core/audio/AudioConnection.java

```diff
@@ -258,28 +266,33 @@ else if (sendHandler == null && sendSystem != null)
 
     private synchronized void setupReceiveSystem()
     {
```

Comment:
Thanks, I did not spot that!

I think it does not make a difference in this case tough, the combined audio thread only gets setup when the udpSocket is not null, so there is nothing to clean up.